lib/spree_gateway/engine.rb

```diff
@@ -28,6 +28,7 @@ class Engine < Rails::Engine
         app.config.spree.payment_methods << Spree::Gateway::Maxipago
         app.config.spree.payment_methods << Spree::Gateway::Migs
         app.config.spree.payment_methods << Spree::Gateway::SpreedlyCoreGateway
+        app.config.spree.payment_methods << Spree::Gateway::PayuLatamGateway
```

Comment:
The Payu Latam gateway only exists in recent versions of ActiveMerchant. Unfortunately Solidus < 1.3 does not support recent versions of ActiveMerchant.

We should only add this gateway if 

```ruby
SolidusSupport.solidus_version > Gem::Version.new("1.3.0")
```